includes/Helpers/GatewayHelper.php

```diff
@@ -89,7 +89,7 @@ public function agreements_field()
 
     public function get_order_by_transaction_crc($crc): ?WC_Order
     {
-        $order = wc_get_orders(['crc' => $crc]);
+        $order = wc_get_orders(['meta_key' => '_crc', 'meta_value' => $crc]);
```

Comment:
Niżej dałbym sprawdzenie czy on czasem nie zwrócił więcej niż jednego order'a i to zalogował.